application/templates/space_profile.html

```diff
@@ -60,10 +60,10 @@ <h2 id="wTitle">{{space.name}}</h2>
             <span class="card-title center-align" id="cardtitle">Contact Info</span>
             <!--<p >Contact Info</p>-->
             {% if space.email %}
-            <a href= "mailto:{{space.email}}" id="tealtext"><i class="material-icons tiny">mail</i> {{space.email | default_if_none:""}}</a><br>
+            <a href= "mailto:#" id="tealtext"><i class="material-icons tiny">mail</i> {{space.email | obfuscate_email | default_if_none:""}}</a><br>
```

Comment:
If you are removing the email from the mailto, I would remove the whole anchor tag. We can add it back in later, if we end up doing that.